scripts/lib/libvirt/compute-config

```diff
@@ -10,14 +10,14 @@ def main():
 
     parser = argparse.ArgumentParser(description="Create Compute Node Config.")
     parser.add_argument("cloud", type=str, help="Name of the Cloud")
-    parser.add_argument("nodecounter", type=str,
+    parser.add_argument("nodecounter", type=int,
```

Comment:
There is the option to wait until the other PR that fix this case is merged, but honestly I like more that is here : ))